SpatialGDK/Documentation/content/multi-layer.md

```diff
@@ -0,0 +1,213 @@
+# Multi-layer development
```

Comment:
Eyes on. Editing in a Gdoc.

Reply:
Looking at this because Rob is sick. Looking for the GDoc link. I can see we need to do some standard-terms-ing and some clarifications.